pages/common/vim.md

```diff
@@ -1,7 +1,7 @@
 # vim
 
 > Vim (Vi IMproved), a command-line text editor, provides several modes for different kinds of text manipulation.
-> Pressing `i` enters edit mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
+> Pressing `i` enters insert mode. `<Esc>` enables the use of Vim commands.
```

Comment:
I think we should mention `normal mode`:
```suggestion
> Pressing `i` enters insert mode. `<Esc>` enters normal mode, which enables the use of Vim commands.
```

Reply:
Is it "normal mode" though, I am not a very experienced VIM user, my knowledge goes as far as knowing what ZZ `:x` and saving commands.

I only know VIM through installing Arch and needing to learn some basics for basic file editing.

Is normal mode correct terminology, I'd call it "command mode"

Reply:
@marchersimon @navarroaxel

Reply:
I would suggest that we should use the official terminology here. Are there any official reference materials that we can look at?

Reply:
Existing as in my edit or before the PR

Reply:
I think the old description might be a bit problematic, because it sounds like you're still in `insert mode`, but pressing `<ESC>` enables you to use Vim commands too. I don't see a problem with the new one.

Reply:
@mfrw

Reply:
I'd say we can apply the suggestion above.

Reply:
```suggestion
> Pressing `i` enters insert mode. `<Esc>` enters command mode.
```

To follow the syntax from the other line before it

Reply:
I actually prefer the longer variant, because it doesn't sound so monotonous.

Reply:
> Is it "normal mode" though, I am not a very experienced VIM user, my knowledge goes as far as knowing what ZZ `:x` and saving commands.
> 
> I only know VIM through installing Arch and needing to learn some basics for basic file editing.
> 
> Is normal mode correct terminology, I'd call it "command mode"

Yep, its officially [normal](http://vimdoc.sourceforge.net/htmldoc/intro.html#vim-modes-intro) mode